server/plugin.go

```diff
@@ -72,7 +72,10 @@ func (p *Plugin) OnActivate() error {
 		return errors.Wrap(appErr, "failed to set profile image")
 	}
 
-	p.API.RegisterCommand(getCommand())
+	if cmdErr := p.API.RegisterCommand(getCommand()); cmdErr != nil {
+		p.API.LogWarn("Error registering Jenkins custom command", "err", cmdErr)
+	}
```

Comment:
Nit pick: It's idiomatic to use just `err` as variable name if there is no naming conflict.